engine/apps/public_api/serializers/integrations.py

```diff
@@ -8,14 +8,47 @@
 from apps.base.messaging import get_messaging_backends
 from common.api_helpers.custom_fields import TeamPrimaryKeyRelatedField
 from common.api_helpers.exceptions import BadRequest
-from common.api_helpers.mixins import NOTIFICATION_CHANNEL_OPTIONS, EagerLoadingMixin
+from common.api_helpers.mixins import (
+    LEGACY_NOTIFICATION_CHANNEL_OPTIONS,
+    NOTIFICATION_CHANNEL_OPTIONS,
+    EagerLoadingMixin,
+)
 from common.jinja_templater import jinja_template_env
 from common.utils import timed_lru_cache
 
 from .integtration_heartbeat import IntegrationHeartBeatSerializer
 from .maintenance import MaintainableObjectSerializerMixin
 from .routes import DefaultChannelFilterSerializer
 
+# PUBLIC_TEMPLATES_FIELDS is map from template name in public api to its db field.
+PUBLIC_TEMPLATES_FIELDS = {
```

Comment:
If we start calling Slack and Telegram "legacy" this should probably be `LEGACY_PUBLIC_TEMPLATES_FIELDS`

Reply:
I'm calling legacy old system, where slack, telegram and other templates are stored in dedicated fields